aten/src/ATen/core/dispatch/backend_fallback_test.cpp

```diff
@@ -104,6 +104,8 @@ TEST(BackendFallbackTest, TestBackendFallbackWithWrapper) {
 TEST(BackendFallbackTest, TestFallthroughBackendFallback) {
   auto m = MAKE_TORCH_LIBRARY_IMPL(aten, TESTING_ONLY_GenericMode);
   m.impl("mul.Tensor", torch::CppFunction::makeFromBoxedFunction<&generic_mode_fallback>());
+  m.impl("abs_", torch::CppFunction::makeNamedNotSupported());
```

Comment:
Since this test has nothing to do with Named, it is probably a good idea to add a little comment saying what exactly we are testing here. Even better would be to test that calling abs_ actually triggers the named not supported exception.